src/app/zap-templates/zcl/data-model/chip/operational-state-rvc-cluster.xml

```diff
@@ -0,0 +1,112 @@
+    <cluster code="0x0061"/>
+    <item name="Stopped"        value="0x00"/>
+    <item name="Running"        value="0x01"/>
+    <item name="Paused"         value="0x02"/>
+    <item name="Error"          value="0x03"/>
+    <item name="SeekingCharger" value="0x40"/>
+    <item name="Charging"       value="0x41"/>
+    <item name="Docked"         value="0x42"/>
+  </enum>
+
+  <enum name="ErrorStateEnum" type="ENUM8">
+    <cluster code="0x0061"/>
+    <item name="NoError"                    value="0x00"/>
+    <item name="UnableToStartOrResume"      value="0x01"/>
+    <item name="UnableToCompleteOperation"  value="0x02"/>
+    <item name="CommandInvalidInState"      value="0x03"/>
+    <item name="FailedToFindChargingDock"   value="0x40"/>
+    <item name="Stuck"                      value="0x41"/>
+    <item name="DustBinMissing"             value="0x42"/>
+    <item name="DustBinFull"                value="0x43"/>
+    <item name="WaterTankEmpty"             value="0x44"/>
+    <item name="WaterTankMissing"           value="0x45"/>
+    <item name="WaterTankLidOpen"           value="0x46"/>
+    <item name="MopCleaningPadMissing"      value="0x47"/>
+  </enum>
+
+  <struct name="OperationalStateStruct">
+    <cluster code="0x0061"/>
+    <item name="OperationalStateID"    type="OperationalStateEnum" optional="false"/>
+    <item name="OperationalStateLabel" type="CHAR_STRING" optional="false" length="64"/>
+  </struct>
+
+  <struct name="ErrorStateStruct">
+    <cluster code="0x0061"/>
+    <item name="ErrorStateID" type="ErrorStateEnum" optional="false" />
+    <item name="ErrorStateLabel" type="CHAR_STRING" optional="false" length="64" isNullable="true"/>
+    <item name="ErrorStateDetails" type="CHAR_STRING" optional="true" length="64" isNullable="true"/>
+  </struct>
+
+  <cluster>
+    <domain>Robots</domain>
+    <name>Robotic Vacuum Operational State</name>
+    <code>0x0061</code>
+    <define>OPERATIONAL_STATE_RVC_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>This cluster supports remotely monitoring and, where supported, changing the operational state of a Robotic Vacuum.</description>
+
+    <globalAttribute side="server" code="0xFFFD" value="1" />
+
+    <attribute side="server" code="0x0000" define="PHASE_LIST"              type="ARRAY" entryType="CHAR_STRING"                                     writable="false" isNullable="true"  optional="false">PhaseList</attribute>
+    <attribute side="server" code="0x0001" define="CURRENT_PHASE"           type="INT8U"                                    min="0x00" max="0x1F"    writable="false" isNullable="true"  optional="false">CurrentPhase</attribute>
+    <attribute side="server" code="0x0002" define="COUNTDOWN_TIME"          type="elapsed_s"                                min="0x00" max="0x3F480" writable="false" isNullable="true"  optional="true">CountdownTime</attribute>
```

Comment:
```suggestion
    <attribute side="server" code="0x0002" define="COUNTDOWN_TIME"          type="elapsed_s"                                min="0" max="259200" writable="false" isNullable="true"  optional="true">CountdownTime</attribute>
```
Not that this matters too much, since we ignore min/max for non-writable attributes.